packaging_automation/update_docker.py

```diff
@@ -96,20 +96,17 @@ def create_directory_if_not_exists(dest_file_name):
         os.makedirs(dir_name)
 
 
-def get_new_changelog_entry(project_version: str, postgres_version: str = ""):
+def get_new_changelog_entry(project_version: str):
```

Comment:
Does this mean that you will manually add a new entry when we want to bump PG versions?

Reply:
Yes because it is not an automated process ad we can update more than one version at once